act-rules-format.bs

```diff
@@ -17,31 +17,27 @@ Introduction {#intro}
 
 
 ACT Rule Types {#rule-types}
 ============================
 
-In accessibility, there are often different technical solutions to make the same type of content accessible. These solutions could be tested in a single rule; however, such a rule tends to be quite complex, making it difficult to understand and maintain. The ACT Rules Format solves this by providing two types of rules:
-
-- <dfn>Atomic rules</dfn> describe how to test a specific type of solution. It contains a precise definition of what elements, nodes or other parts of a web page are to be tested, and when those elements are considered to fail the rule. These rules should be kept small and atomic. Meaning that atomic rules test a single "failure condition", and do so without using results from other rules.
+In accessibility, there are often different technical solutions to make the same type of content accessible. Multiple solutions could be tested in a single rule; however, such a rule tends to be quite complex, making it difficult to understand and maintain. The ACT Rules Format solves this by providing two types of rules:
 
-- <dfn>Composite rules</dfn> describe how results from atomic rules should be used to decide if an accessibility requirement was failed. If there are multiple failures necessary for an accessibility requirement to fail, each "type" of failure should be written as separate atomic rules, and the logic for determining the ´pass´ or ´fail´ for the accessibility requirement, across the atomic rules, should be handled by a composite rule.
+- <dfn>Atomic rules</dfn> describe how to test a specific type of solution. It contains a precise definition of what elements, nodes or other parts of a [=test subject=] are to be tested, and when those elements are considered to fail the rule. These rules are to be kept small and *atomic*. Meaning that atomic rules test a single "failure condition", and do so without using the [=outcome=] of other rules.
 
-The separation between atomic rules and composite rules creates a division of responsibility. Atomic rules test if web content is correctly implemented in a particular solution. Composite rules test if a combination of `pass` outcomes from atomic rules is sufficient for an accessibility requirement not to `fail`. Not all atomic rules have to be part of a composite rule. Atomic rules only have to be part of a composite rule if failing that atomic rule does not directly fail [Accessibility Requirements](#structure-accessibility-requirements). An atomic rule MAY be part of multiple composite rules.
+- <dfn>Composite rules</dfn> describe how the [=outcome=] of [=atomic rules=] is used to decide if an accessibility requirement is (non-)conformant. If this involves multiple "failure conditions", each of these is provided in a separate atomic rule. The logic for determining (non-)conformance to the accessibility requirement according to outcome of the individual the atomic rules is specified in the composite rule.
```

Comment:
- "decide if an accessibility requirement is (non-)conformant" does not make sense. Maybe "decide if a test subject conforms to an accessibility requirement"?
- Same with "logic for determining (non-)conformance to the accessibility requirement" -- needs "test subject" -- "logic for determining conformance of the test subject to the accessibility requirement"
- *BUT* if you define composite rules that way, then each composite rule *MUST* be related to an accessibility requirement. However, the requirement later on in section 7 is only a SHOULD requirement. You need to either change that requirement or this definition of composite rules.